specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -553,6 +556,28 @@
         }
       }
     },
+    "ClientGroupInfo": {
```

Comment:
It's strongly preferred to add new configurable properties in a new API version. The issue is when a user updates the new properties, and another user does a GET->PUT with an older SDK. The new properties would be omitted from the request body from the downlevel SDK

Reply:
> It's strongly preferred to add new configurable properties in a new API version. The issue is when a user updates the new properties, and another user does a GET->PUT with an older SDK. The new properties would be omitted from the request body from the downlevel SDK

Thanks for your comment. This new properties is not mandatory. And I agree with your opinion.